### ISSUE TITLE
Add ability to resend verification email

### ISSUE THREAD

**Issue by kazlauskis** [NERC-CEH/fit-count-app#82]
If the user is trying to upload a survey with an unverified account we currently show a warning to click on the verification email first. It can happen that the user has registered a while ago and so the verification email might have been deleted from the email account by then.

<img width="402" alt="Screenshot 2021-05-18 at 12 13 57" src="https://user-images.githubusercontent.com/7383162/118642259-0b453700-b7d3-11eb-937e-829aada699b8.png">

From #78

**Comment by kitenetter** [comment]
If there is any way of priorotising this I would be really grateful. We have about 100 blocked users at the moment. I just tried to do a bulk unblock but that doesn't seem to have worked: "The application did not respond in time."

@CCarvell for info.

**Comment by CCarvell** [comment]
@kazlauskis Martin is on leave this week but this is an urgent issue that needs to be resolved quickly if possible please.
I'm not sure I can do much to help?
I was going to put a short Tweet out to report that we have having technicall issues with app registration but would like to know a bit more first, and also cannot deal with a flood of enquiries myself today!

@kazlauskis let me know if a quick chat would help?

**Example email to poms from Website Feedback:**
 I have tried to reset my FIT count app but keep  getting no response. I had posted several datasets ok but these have been wiped out.
Please can you sort this out?
With thanks
Kit Welchman
P.S. i think I now have 2 accounts - kit4@totalise.co.uk and welchmanchristopher@gmail.com Neither of which work for FITcount.
I have tried  changing my password severaltimes

**Comment by kazlauskis** [comment]
Yes, the website might be failing to send the registration emails so we should investigate that area first.

**Comment by CCarvell** [comment]
OK @kazlauskis glad you have seen this! Do you mean yes to a quick chat or yes to investigating the website emails issue yourself now?

**Comment by kazlauskis** [comment]
Sorry, I am in meetings all day today but I am free after 6pm. I wouldn't have the capacity to investigate the server email issues now, but Biren has set it up so he should be able to look into this.

**Comment by BirenRathod** [comment]
@kazlauskis & @CCarvell Email works fine but this is to do with users already registered in the warehouse. so they don't email if they have already registered in the warehouse.

**Comment by CCarvell** [comment]
@kitenetter @BirenRathod I have responded with several points under #114 on the PoMS website GitHub - this all seems to be linked. Also this just happened:
I just logged out of my app account and back in again.
Now I cannot see any of my Uploaded surveys showing on the app - it says "No uploaded surveys". This seems to be the same as for Kit Welchman above who said "I had posted several datasets ok but these have been wiped out." and " i think I now have 2 accounts"
Do we have a major issue with the app and Warehouse account/s registration here?
